assets/js/src/core/utils/route.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
hmm, I think this file would be better located in the general app folder /app/api/pimcore.
For me it's something more essential for the app, rather than a utility to use when needed.

What do you think?